Source/Schema.NET/ValuesJsonConverter.cs

```diff
@@ -307,18 +309,15 @@ private static bool TryProcessTokenAsType(JsonReader reader, Type targetType, ou
                         enumString = valueString;
                     }
 
-                    try
+                    if (Enum.TryParse(targetType, enumString, out result))

```

Comment:
`Enum.TryParse` instead of try/catch. This could also be in another PR.

Reply:
Don't know why we didn't do this sooner!

Reply:
Exactly! Performance win too!

Reply:
I think I found out why - that method doesn't exist in .NET Standard 2.0 😣

Reply:
Done! Created an `EnumHelper` static class with a `TryParse` method that copies the signature for `Enum.TryParse`.